packages/frontend/src/components/KeyboardShortcutHint.tsx

```diff
@@ -12,6 +12,26 @@ const keySymbols: { [key: string]: string } = {
   ArrowUp: '↑',
 }
 
+// this would be the approach to show users how to access
+// certain keyboard codes according to their layout
```

Comment:
"Would be", but why it's not the case already? Hard to understand without context,

Reply:
This was just a comment related to the discussion we had before (in chat). I will remove it before merging